app/services/courses/copy_to_provider_service.rb

```diff
@@ -5,8 +5,8 @@ def initialize(sites_copy_to_course:, enrichments_copy_to_course:)
       @enrichments_copy_to_course = enrichments_copy_to_course
     end
 
-    def execute(course:, new_provider:)
-      return nil unless course.rollable?
+    def execute(course:, new_provider:, force: false)
+      return nil unless course.rollable? || force
```

Comment:
This allows for any course to be rollover, @dankmitchell, is that the desired behaviour?

Reply:
@defong we want the ability to rollover courses manually as well as rolling over courses that has draft enrichments

Reply:
For clarity; we need to re-run the rollover command WITHOUT "force" to do a sweep of all the "published with draft enrichments" courses.